Documentation/GeneralConventions/_guides-simple.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<guides
+        xmlns="https://www.phpdoc.org/guides"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="https://www.phpdoc.org/guides ../vendor/phpdocumentor/guides-cli/resources/schema/guides.xsd"
+        links-are-relative="true"
+>
+    <project
```

Comment:
I would like to see not a docs team manuals but an extension manual example. This will be the use case in 99% of use cases, I assume.

release and version should hold then real values, not main.